bundles/sirix-core/src/main/java/org/sirix/access/xml/XmlResourceStore.java

```diff
@@ -25,13 +25,6 @@
  */
 public final class XmlResourceStore extends AbstractResourceStore<XmlResourceManager> {
 
-  /**
-   * Default constructor.
-   */
-  public XmlResourceStore() {
-    super(new ConcurrentHashMap<>(), new User("admin", UUID.randomUUID()));
```

Comment:
We're already decoupling this concept of "default user" from the resource store implementation :)